test/drbgtest.c

```diff
@@ -1264,7 +1264,8 @@ static const size_t crngt_num_cases = 6;
 
 static size_t crngt_case, crngt_idx;
 
-static int crngt_entropy_cb(unsigned char *buf, unsigned char *md,
+static int crngt_entropy_cb(OPENSSL_CTX *ctx, unsigned char *buf,
```

Comment:
Hmm - is this a backwards compatibility breaking change?

Reply:
No. This callback is only exposed because drbgtest.c includes rand_lcl.h